tests/utilities/namespace_mock.py

```
@@ -26,7 +26,9 @@
 
 """
 
+from typing import Any
+
 
 class Namespace:
-    def __init__(self, **kwargs):
+    def __init__(self, **kwargs: Any) -> None:
```

Ruff (0.14.4)

Dynamically typed expressions (typing.Any) are disallowed in `**kwargs`

(ANN401)